meetings/2024-02/February-6.md

```diff
@@ -0,0 +1,1371 @@
+# 6th Feb 2024 100th TC39 Meeting
+
+-----
+
+Delegates: re-use your existing abbreviations! If you’re a new delegate and don’t already have an abbreviation, choose any three-letter combination that is not already in use, and send a PR to add it upstream.
+
+You can find Abbreviations in delegates.txt
+
+**Attendees:**
+| Name                   | Abbreviation | Organization      |
+| ---------------------- | ------------ | ----------------- |
+| Chris de Almeida       | CDA          | IBM               |
+| Linus Groh             | LGH          | Bloomberg         |
```

Comment:
the attendee tables are removed before publishing

Reply:
that's not true

Reply:
huh, you're right, i could have sworn they were, but i randomly clicked on some older ones and they're there. sorry for the noise.